brooklyn-dist/vagrant/src/main/vagrant/servers.yaml

```diff
@@ -0,0 +1,47 @@
+---
+default_config:
+    check_newer_vagrant_box: true
+    run_os_update: true
+servers:
+  - name: brooklyn
+    box: ubuntu/vivid64
+    ram: 2048
+    cpus: 4
+    ip: 10.10.10.100
+    shell:
+      env:
+        BROOKLYN_VERSION: 0.9.0-SNAPSHOT
+      cmd:
+        - sudo sh -c 'export DEBIAN_FRONTEND=noninteractive; apt-get install --yes openjdk-8-jre-headless'
+        - curl -s -S -J -O -L "https://www.apache.org/dyn/closer.cgi?action=download&filename=brooklyn/apache-brooklyn-${BROOKLYN_VERSION}/apache-brooklyn-${BROOKLYN_VERSION}-bin.tar.gz"
```

Comment:
it will need testing but we should be able to do the same to script the vagrant download, just replace the right `dist` mention with `vagrant` (ie not the `brooklyn-dist`!).

BTW i recall problems in the past with `closer.cgi` which prompted the use of `closer.lua` instead -- not sure how good that advice is but we've had no issues with the latter since the switch.  it seems to take the same arguments, which i didn't know about (i only knew about the redirect to the mirror, which our download page points you to ... probably worth updating the download page not to redirect you there unless you want to, and we can remove the "Sorry" message!); i think you're right it's better to use that, so for official releases instead do:

    https://www.apache.org/dyn/closer.lua?action=download&filename=brooklyn/apache-brooklyn-${BROOKLYN_VERSION}/apache-brooklyn-${BROOKLYN_VERSION}-bin.tar.gz

and `s/bin/vagrant` of course elsewhere.  (i think the `-bin` is the same as the `-dist` and vagrant would be the `-vagrant` in both places ... but not entirely sure.)